### PR TITLE
[js] try to fix `eslint` issue on `git commit`

### PR DESCRIPTION
**Pull request by zamboney**

### CODE REVIEW

**Comment by kyryloonufriiev** [comment]
@zamboney What does it do?

**Comment by zamboney** [comment]
> Ah, you just absolutize the path of staged files, right?

yep 🎉

**Comment by kyryloonufriiev** [comment]
I think you should not change the hook file, but instead, simply change the script in `package.json`. The idea is to be able to run exactly the same command manually as the git hook will run.

**Comment by zamboney** [comment]
> I think you should not change the hook file, but instead, simply change the script in `package.json`. The idea is to be able to run exactly the same command manually as the git hook will run.

i was able to fix it by passing '../' to the `--line-prefix`

**Comment by kyryloonufriiev** [comment]
I can agree with the absolute path. But I don't understand what `../` solves.
Whatever we have now works fine for me. Are you sure it will continue working after this pr?

**Comment by zamboney** [comment]
The absolute path was just a symptom of the real problem, which is that the combination of using `npm script` and `git diff` added a duplication of `js` in the path. this solution fixes that by changing the path to one directory up

**Comment by kyryloonufriiev** [comment]
Why don't I have this problem then?

**Comment by zamboney** [comment]
@kyryloonufriiev this is strange, can you check with my changes and see if you have an issue? Because I think that @SophieDePaula and I getting the same issue.

**Comment by SophieDePaula** [comment]
My issue was that I merged `master` into my branch because I had conflicts which brought many changes from master that eslint checked. Then I had to run `yarn` in one of the packages that was changed because I got an error that one of the `eslint` packages was not found there, but I couldn't install all the dependencies in that package because that package had a dependency that was not published yet.

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]

File: js/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "cli-table3": "^0.6.3",
     "eslint": "^8.32.0",
     "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-mocha-no-only": "^1.1.1",
```
[the remark this answers is not in the document]
Comment (zamboney) [reply js/package.json:23]: `eyes-cypress` use it

[the remark this answers is not in the document]
Comment (kyryloonufriiev) [reply js/package.json:23]: So it is not related to the repo level modules/configs

[the remark this answers is not in the document]
Comment (zamboney) [reply js/package.json:23]: nope.

**Comment by zamboney** [comment]
after i clone a new repo and test it on (without `git worktree`) all work as expected so this is a `git worktree` issue. i will try to solve it on my end but i'm closing this pr

**Pull request closed without merging**